index.js

```diff
@@ -46,6 +50,11 @@ document.getElementById('submit_button').addEventListener('click', function () {
     user_feedback_elem.style.display = 'block'
 })
 
+document.getElementById('next_question_button').addEventListener('click', function () {
+    get_new_question()
+    document.getElementById('next_question_button').disabled = true
```

Comment:
This may be the issue of bugs. Stops us being able to progress.

Reply:
done!